File: Beginner/Positive_Numbers_1060/main.go
package main

import "fmt"

const amountNumbers = 6

func main() {
	var actualNumber float64
	var amountPositiveNumbers int
	for count := 0; count < amountNumbers; count++ {
		fmt.Scan(&actualNumber)
		if actualNumber >= 0 {
			amountPositiveNumbers += 1
		}
	}
	fmt.Printf("%d valores positivos\n", amountPositiveNumbers)
}

File: Beginner/Positive_Numbers_1060/Main.java
package Beginner.Positive_Numbers_1060;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int amountPositiveNumbers = 0;
        double actualNumber;
        for (int count = 0; count < 6; count ++) {
            actualNumber = sc.nextDouble();
            if (actualNumber >= 0) {
                amountPositiveNumbers += 1;
            }
        }
        System.out.printf("%d valores positivos\n", amountPositiveNumbers);
        sc.close();
    }
    
}
